metabase/metabase
# Add a way to unarchive timelines and delete them permanently

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Adds a link to `View archived timelines` in the timelines list and details modals
- Shows archived timelines with a menu to unarchive and delete them

How to test:
- Create a timeline in a collection
- Archive the timeline
- Go to `Archived timelines` modal via the header menu
- Unarchive the timeline. Make sure it's possible to undo this action.
- Delete the timeline

<img width="646" alt="Screenshot 2022-03-24 at 18 03 41" src="https://user-images.githubusercontent.com/8542534/159947240-4a2f0030-b96e-49da-80b6-442b391a7b58.png">
<img width="651" alt="Screenshot 2022-03-24 at 18 04 32" src="https://user-images.githubusercontent.com/8542534/159947264-b4df7fb0-6a50-4b9e-9e7a-84ebea30a391.png">
<img width="649" alt="Screenshot 2022-03-24 at 18 04 38" src="https://user-images.githubusercontent.com/8542534/159947271-57e9e458-e30b-4094-b757-cee36195a23b.png">
